src/components/AccordionTable/AccordionTableBasic.part.tsx

```diff
@@ -186,16 +220,40 @@ export class AccordionTableBasic<T> extends React.Component<AccordionTableProps<
     );
   }
 
+  private getGroupByValue = (rowData: any) => {
```

Comment:
why `any`?

Reply:
If I use `T` instead I have the problem with the check `if (rowValue || rowValue === 0)`. TypeScript tells me that `rowValue` won't ever be a number and therefore the expression `rowValue === 0` will always be false.